### ISSUE TITLE
Alexnet high accuracy

### ISSUE THREAD

**Issue by zeweiding-kk** [IvoryCandy/pytorch-cifar10#6]
I trained the "Alexnet" in another implementation and got very high accuracy, around 85%. 

The main differences between the two implementations are, 

1. random cropping in train_transform "transforms.RandomCrop(32, padding=4)", 
2. SGD optimizer with step-wise learning rate [100 150 180] (total 200 epochs) and an initial learning rate 0.1. 

Is this high accuracy possible?
